test/server/lib/validation/index.test.js

```diff
@@ -1,9 +1,8 @@
 const logger = require(`${serverRoot}/config/winston`);
-const errorManifest = require(`${serverRoot}/lib/errors/error_manifest`).validation;
 const Validator = require(`${serverRoot}/lib/validation`);
 const validator = new Validator();
 
-describe.skip('server/lib/validation/applications', () => {
+describe('server/lib/validation/applications', () => {
```

Comment:
The description for the test suite should try to describe the module under test as closely as possible. For example, here, I would have used: `describe('server/lib/validation/index', () => { ...`